torch/_C/_distributed_rpc.pyi

```diff
@@ -0,0 +1,161 @@
+from typing import Tuple, Dict, Optional, List, Any, overload
+from datetime import timedelta
+import enum
+import torch
+from ._autograd import ProfilerConfig, ProfilerState
+from ._distributed_c10d import ProcessGroup, Store
+
+_DEFAULT_NUM_SEND_RECV_THREADS: int
+_DEFAULT_INIT_METHOD: str
+_DEFAULT_NUM_WORKER_THREADS: int
+_UNSET_RPC_TIMEOUT: float
+_DEFAULT_RPC_TIMEOUT_SEC: float
+# distributed/rpc/rpc_agent.h
```

Comment:
nit: please add empty line between comments and previous line?

Reply:
Right, will update.